packages/medusa-react/src/hooks/admin/payment-collections/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./queries"
+export * from './mutations'
```

Comment:
todo: Should use double quotes `"`
```suggestion
export * from "./mutations"
```